### ISSUE TITLE
Helpers: Task validation feature

### ISSUE THREAD

**Issue by LajosCseppento** [Yachting-Club-CERN/ycc-app#48]
The contact should be able to validate/invalidate the captain/helpers for the shift.

Discussed with Gijs:

```
Last year [2022] we discussed something similar on the committee, where Michal said that for data privacy we should not put in our systems if someone was a "no show", but rather just remove the person from the shift (i.e., like if that the person never signed up).

For now I propose the following:

* tasks can be validated by the person who is the contact
* when you validate the task, the system saves when it was validated and by whom, further sign ups are not possible

To minimise how much things I need to implement for now you would need to contact me if you want to:
* Remove a helper from the task (even after validation, e.g., person in the end did not help)
* "Unvalidate" the task, i.e., put back in the state where it is not validated

So basically validation means = done, and of course we could filter which tasks are NOT done.
```
